doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,70 @@
+
+# PaddlePaddle Client
+PaddlePaddle clinet is a command line tool, before startting a cluster train, you need to install it on your latop, you can sumit a cluster train job looks like:
+
+```bash
+paddle k8s job <job-name>
```

Comment:
Should be `paddle k8s_train ...`, use a single subcommand or use the `kubectl` style: `paddle submit k8s_job -f [job.yaml]`